ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/Impl.hs

```diff
@@ -324,17 +324,18 @@ putBlockImpl env@VolatileDBEnv{ maxBlocksPerFile, tracer }
                     ++ "Current write file not found in Index.")
             (Index.lookup currentWriteId currentMap)
         fileBlockInfo = FileInfo.mkFileBlockInfo (BlockSize bytesWritten) bbid
-        fileInfo' = FileInfo.addBlock bslot currentWriteOffset fileBlockInfo fileInfo
+        AbsOffset offset = currentWriteOffset
+        fileInfo' = FileInfo.addBlock bslot offset fileBlockInfo fileInfo
         currentMap' = Index.insert currentWriteId fileInfo' currentMap
         internalBlockInfo' = InternalBlockInfo {
             ibFile         = currentWritePath
-          , ibBlockOffset  = currentWriteOffset
+          , ibBlockOffset  = offset
```

Comment:
What if you change `BlockOffset` (in VolatileDB.Types) to `AbsOffset` too? I'm just wondering if that's a good change.